lib/server/services/runDetector/getRunDetector.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const { utilities: { QueryBuilder } } = require('../../../database');
+const RunDetectorsRepository = require('../../../database/repositories/RunDetectorsRepository.js');
+const { utilities: { TransactionHelper } } = require('../../../database');
+
+/**
+ * Find and return an run detector model by its id
+ *
+ * @param {string} runNumber the runNumber of the run to find
+ * @param {string} detectorId the id of the detector to find
+ * @param {function|null} qbConfiguration function called with the run detector find query builder as parameter to add specific configuration to
+ *     the query
+ * @param {Object|null} [transaction] optionally the transaction in which one the log creation is executed
```

Comment:
It may be wrong in my own code, but you can specify the default value of optional code in the square brackets `@param {function|null} [qbConfiguration=null]`

Reply:
done